### PR TITLE
API for testing destination setup

### PR DESCRIPTION
**Pull request by arajguha**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: util/customTransforrmationsStore-v1.js [hunk 1]
```diff
@@ -5,9 +5,9 @@ const stats = require("./stats");
 const transformationCache = {};
 const libraryCache = {};
 
-const CONFIG_BACKEND_URL =
-  process.env.CONFIG_BACKEND_URL || "https://api.rudderlabs.com";
-
+// const CONFIG_BACKEND_URL =
+//   process.env.CONFIG_BACKEND_URL || "https://api.rudderlabs.com";
+const CONFIG_BACKEND_URL = "http://localhost:5000";
```
Comment (sanpj2292) [inline util/customTransforrmationsStore-v1.js:10]: Can we revert this change ?

File: util/customTransforrmationsStore.js [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
 const logger = require("../logger");
 const stats = require("./stats");
 
 const myCache = new NodeCache();
 
-const CONFIG_BACKEND_URL =
-  process.env.CONFIG_BACKEND_URL || "https://api.rudderlabs.com";
+// const CONFIG_BACKEND_URL =
+//   process.env.CONFIG_BACKEND_URL || "https://api.rudderlabs.com";
+const CONFIG_BACKEND_URL = "http://localhost:5000";
```
Comment (sanpj2292) [inline util/customTransforrmationsStore.js:10]: Can we revert this change ?

File: util/customTransforrmationsStore-v1.js (hunk 1 above)
Comment (arajguha) [reply util/customTransforrmationsStore-v1.js]: yes reverted for now. Although it needs to be set to the dev config-be URL for the time being, since its going to be merged to develop first

**Review by saikumarrs — APPROVED** [review]

**Comment by arajguha** [comment]
Moved to https://github.com/rudderlabs/rudder-transformer/pull/856
